package.json

```diff
@@ -17,10 +17,16 @@
     "lint:css": "stylelint 'src/**/*.css'",
     "lint:css:fix": "stylefmt --recursive src/",
     "lint:staged": "lint-staged",
+    "format": "npm-run-all -p format:js format:css",
+    "format:js": "prettier --write ./src/**/*.js",
+    "format:css": "prettier --write ./src/**/*.css",
     "deps": "npm-check -s",
     "deps:update": "npm-check -u",
     "prepublishOnly": "npm run build"
   },
+  "prettier": {
+    "trailingComma": "all"
+  },
   "lint-staged": {
```

Comment:
As lint-staged is already being used, I believe Prettier can be [added here](https://prettier.io/docs/en/precommit.html#option-1-lint-staged-https-githubcom-okonet-lint-staged) (alongside `eslint --fix`).

Reply:
Done.